### PR TITLE
LPD-1261 Replace cards by cards with radio buttons

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/css/_AddFragmentModal.scss [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
-	flex-grow: 1;
+.fragment-type-cards {
+	gap: 12px;
 
-	&:first-child {
-		margin-right: 12px;
-	}
+	.fragment-type-card {
+		flex-basis: 0;
+		flex-grow: 1;
```
Comment (victorg1991) [inline modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/css/_AddFragmentModal.scss:6]: We can just use flex:1 here :P

**Comment by victorg1991** [comment]
Just two little details and this is good to go :D

**Comment by veroglez** [comment]
ci:forward

**Pull request closed without merging**